src/backend/didrequest.c

```diff
@@ -180,12 +180,15 @@ const char *DIDRequest_Sign(DIDRequest_Type type, DIDDocument *document, DIDURL
 
 int DIDRequest_Verify(DIDRequest *request)
 {
-    return DIDDocument_Verify(request->doc, &request->proof.verificationMethod,
-            (char*)request->proof.signature, 4,
-            request->header.spec, strlen(request->header.spec),
-            request->header.op, strlen(request->header.op),
-            request->header.prevtxid, strlen(request->header.prevtxid),
-            request->payload, strlen(request->payload));
+    //todo: if(request->doc) is for deacativated without doc.
```

Comment:
return !request->doc ? 0 : DIDDocument_Verify(....)

Reply:
or filter out the invalid scenario with "!request->doc"
if (!request->doc)
     return 0

return DIDDocument_Verify(....);